ibraheemosule/skoolizy
# Za - add-account-details

# Why this PR?
- To incorporate the user's personal details to the platform

# What does this PR do?
- It adds the user's personal details to the Profile section.

# How to test your work?
once you login,  navigate to the profile page and the user details should be reflected in the appropriate sections

# Pages to see your changes?
-  /my-profile/personal-information
-/my-profile/contact-information
